File: Python/Medium/Sorting/2545__Sort_the_Students_by_Their_Kth_Score.py
def sortTheStudents(score: list[list[int]], k: int) -> list[list[int]]:
    return sorted(score, key=lambda x: x[k], reverse=True)


score = [[10, 6, 9, 1], [7, 5, 11, 2], [4, 8, 3, 15]]
k = 2
print(sortTheStudents(score, k))

File: PHP/Medium/Sorting/2545__Sort_the_Students_by_Their_Kth_Score.php
<?php
function sortTheStudents($score, $k)
{
    usort($score, function ($a, $b) use ($k) {
        return $b[$k] <=> $a[$k];
    });
    return $score;
}

$score = [[10, 6, 9, 1], [7, 5, 11, 2], [4, 8, 3, 15]];
$k = 2;

var_dump(sortTheStudents($score, $k));